bff-service/src/services/cache-request.service.ts

```diff
@@ -0,0 +1,77 @@
+import { Injectable } from '@nestjs/common';
+import { Request } from 'express';
+import { ConfigService } from '@nestjs/config';
+import * as redis from 'redis';
+import { IncomingHttpHeaders } from 'http';
+
+export interface ResponseCache {
+  status: number;
+  data: any;
+}
+
+@Injectable()
+export class CacheRequestService {
+  private redisClient = redis.createClient(
+    +this.configService.get('REDIS_PORT', 6379),
```

Comment:
Maybe it would be better to hide in .env?

Reply:
Don`t really got this, sorry. This code already looks for REDIS_PORT env variable and if it`s not there it takes 6379 as a default.

Reply:
Oh, got it! Ignore my comment :)